framework/src/utils/TransitionLayerTools.C

```diff
@@ -0,0 +1,614 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "TransitionLayerTools.h"
+#include "MooseMeshUtils.h"
+#include "MooseMesh.h"
+#include "MeshGenerator.h"
+#include "MooseError.h"
+
+// libMesh includes
+#include "libmesh/mesh_generation.h"
+#include "libmesh/unstructured_mesh.h"
+#include "libmesh/point.h"
+#include "libmesh/elem.h"
+#include "libmesh/node.h"
+#include "libmesh/face_tri3.h"
+#include "libmesh/face_quad4.h"
+
+// C++ includes
+#include <cmath> // provides round, not std::round (see http://www.cplusplus.com/reference/cmath/round/)
```

Comment:
@miaoyinb Working through this on M1 today, but I had a quick question. Is this include still necessary? It doesn't look like it's used here or in the reactor module.

Reply:
Thanks, @cticenhour.
I remember that I added this include for `elementsCreationFromNodesVectorsQuad()` here.

Reply:
It doesn't look like its used any longer for `elementsCreationFromNodesVectorsQuad`, so we should clean this up. Will add to a more formal review.